website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -686,17 +722,15 @@ Use the `alias` field to select an EKS-optimized AMI family and version. Family
 
 The version string can be set to `latest`, or pinned to a specific AMI using the format of that AMI's GitHub release tags.
 For example, AL2 and AL2023 use dates for their release, so they can be pinned as follows:
-```
+```yaml
 alias: al2023@v20240703
 ```
 Bottlerocket uses a semantic version for their releases. You can pin bottlerocket as follows:
-```
+```yaml
 alias: bottlerocket@v1.20.4
 ```
 The Windows family does not support pinning, so only `latest` is supported.
 
-An `alias` is mutually exclusive and may not be specified with any other terms.
```

Comment:
Why did we drop the line about alias?

Reply:
It was relocated to the actual API description